health.go

```diff
@@ -242,48 +243,63 @@ type Partition struct {
 	InodeFree    uint64 `json:"inode_free,omitempty"`
 }
 
+// NetInfo contains information about a network inerface
+type NetInfo struct {
+	NodeCommon
+	Interface       string `json:"interface,omitempty"`
+	Driver          string `json:"driver,omitempty"`
+	FirmwareVersion string `json:"firmware_version,omitempty"`
+}
+
 // Partitions contains all disk partitions information of a node.
 type Partitions struct {
 	NodeCommon
 
 	Partitions []Partition `json:"partitions,omitempty"`
 }
 
-func getDeviceModel(partDevice string) (string, error) {
-	var model string
-
+func getDeviceInfo(partDevice string) (model string, revision string, err error) {
```

Comment:
Do we want to create a struct for `DeviceInfo` itself so that latest we can add more attributes if needed?

Reply:
@shtripat done, PTAL